intro-HPC/ch_running_jobs_with_input.tex

```diff
@@ -540,7 +578,7 @@ \subsection{Check your quota}
 3.5M %\homedir/primes.txt%
 \end{prompt}
 
-We also want to mention the \emph{tree} command, as it also provides an easy
+We also want to mention the \texttt{tree} command, as it also provides an easy
```

Comment:
we should probably also mention you shouldn't really use this command all that often, like, not in job scripts etc. since it can put a heavy strain on the file system. Same with `du` actually.